blueman/plugins/applet/DiscvManager.py

```diff
@@ -56,9 +56,12 @@ def on_manager_state_changed(self, state: bool) -> None:
             self.update_menuitems()
 
     def on_update(self) -> bool:
-        self.time_left -= 1
-        self.item.set_text(_("Discoverable… %ss") % self.time_left)
         self.item.set_sensitive(False)
+        if self.time_left >= 1:
```

Comment:
Why not for 0?

Reply:
Feel free to change :)

In realty, it could be <0. So compare:
`Discoverable... 2s`
`Discoverable... 1s`
`Discoverable... 0s`
`Discoverable...`
vs
`Discoverable... 2s`
`Discoverable... 1s`
`Discoverable...`